app/assets/stylesheets/application.scss

```diff
@@ -7,3 +7,51 @@
 @import "base/base";
 
 @import "layout";
+
+div.nav {
+
+	ul.menu {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Line should be indented with spaces, not tabs<br>Avoid qualifying class selectors with an element.